File: Cpp/771_Jewels_and_Stones.cpp
class Solution {
public:
    int numJewelsInStones(string J, string S) {
        int num = 0;
        char *j = (char *) J.c_str();
        char *s = (char *) S.c_str();
        for (int i = 0; i < J.size(); i++,j++)
            for (int k = 0; k < S.size(); k++)
                if (*j == s[k]) num++;
        return num;
    }
};

File: Python/771_Jewels_and_Stones.py
class Solution:
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        # return sum(s in J for s in S)
        # return sum(map(S.count, J))
        return sum(map(J.count, S))


        
